tests/utils.py

```diff
@@ -49,3 +51,99 @@
     '_id': 'yes! yes! yes!',
     'count': 0
 }
+
+
+TEST_SCHEMA = update_schema(BASEXMLSCHEMA, {
+    "properties": {
+        "title1": "//dc:title/node()",
+        "title2": ["//dc:title/node()", lambda x: x.lower()],
+        "title3": ["//dc:title/node()", "//dc:title/node()", lambda x, y: x + y.lower()]
```

Comment:
I'd rather these be tuples: ((args, kwargs), callable) where args or kwargs could be none and the first element would not necessarily be a tuple.